app/controllers/BannerController.scala

```diff
@@ -111,7 +111,7 @@ class BannerController @Inject()(
           for {
             minDetails <- minConnector.getMinimalPspDetails(pspId)
             _ <- emailConnector.sendEmail(SendEmailRequest.apply(
-              to = List("david.saunders@digital.hmrc.gov.uk"),
+              to = List("pdc-pensions-g@digital.hmrc.gov.uk"),
```

Comment:
If we leave this here then when we test in QA environment the test emails will be sent to this email address, which the stakeholders probably won't appreciate. I suggest getting this email address from the application.conf file and then overriding with this email address in the app config production repository.